instructors.md

```diff
@@ -3,28 +3,126 @@ layout: page
+    so collect learners' names during that session
+    and create a repository for them to share
+    with their names as both their IDs and their passwords.
 
-## [Conflicts](03-conflict.html)
+*   If your learners are advanced enough to be comfortable SSH,
+    tell them they can use keys to authenticate on GitHub instead of passwords,
+    but don't try to set this up during class:
+    it takes too long,
+    and is a distraction from the core ideas of the lesson.
 
-FIXME
+*   Be very clear what files learners are to edit
+    and what user IDs they are to use
+    when giving instructions:
+    it is common for them to edit the file the instructor is working on
+    rather than their own,
+    or to use the instructor's user ID and password when committing.
 
-## [Open Science](04-open.html)
+*   Be equally clear *when* they are to edit things:
+    it's also common for someone to edit the file the instructor is editing
+    and commit changes while the instructor is explaining what's going on,
+    so that a conflict occurs when the instructor comes to commit the file.
 
-FIXME
+*   If some learners are using Windows,
+    there will inevitably be issues merging files with different line endings.
+    (Even if everyone's on some flavor of Unix,
+    different editors may or may not add a newline to the last line of a file.)
+    Take a moment to explain these issues,
+    since learners will almost certainly trip over them again.
+
+*   We don't use a Git GUI in these notes because
+    we haven't found one that installs easily and runs reliably on the three major operating systems,
+    and because we want learners to understand what commands are being run.
+    That said,
+    instructors should demo a GUI on their desktop at some point during this lesson
+    and point learners at [this page](http://git-scm.com/downloads/guis).
+
+*   Instructors should also show learners graphical diff/merge tools like
+    [DiffMerge](https://sourcegear.com/diffmerge/).
+
+*   When appropriate,
+    explain that we teach Git rather than CVS, Subversion, or Mercurial
+    primarily because of GitHub's growing popularity:
+    CVS and Subversion are now seen as legacy systems,
+    and Mercurial isn't nearly as widely used in the sciences right now.
+
+## Text Editor
+
+We suggest instructors and students use `nano` as the text editor for this lessons because:
+
+*   it runs in all three major operating systems,
+*   it runs inside the shell (switching windows can be confusing to students), and
+*   it has shortcut help at the bottom of the window.
+
+Please point out to students during setup
+that they can and should use another text editor
+if they're already familiar with it.
+Below you will find some tips that could help solving problems
+when using other editors.
+
+### Gedit
+
+You should use
+
+~~~ {.bash}
+$ git config --global core.editor 'gedit --standalone'
+~~~
+
+to avoid this error occurring
```

Comment:
Maybe

    to avoid

    ~~~~ {.bash}
    $ git commit
    ~~~
    ~~~ {.error}
    Aborting commit due to empty commit message.
    ~~~

    that occurs